### ISSUE TITLE
bug in mal_id configuration

### ISSUE THREAD

**Issue by orenmazor** [Kometa-Team/Kometa#1253]
### Version Number

1.18.1

### What branch are you on?

master

### Describe the Bug

I just checked out a brand new copy of PMM and with a fresh templated config got this error:

```
| Playlist File Loaded Successfully                                                                  |
| Traceback (most recent call last):                                                                 |
|   File "/Users/orenmazor/Projects/Plex-Meta-Manager/modules/config.py", line 595, in __init__      |
|     self.Convert = Convert(self)                                                                   |
|                    ^^^^^^^^^^^^^                                                                   |
|   File "/Users/orenmazor/Projects/Plex-Meta-Manager/modules/convert.py", line 25, in __init__      |
|     self._mal_to_anidb[int(ids["mal_id"])] = anidb_id                                              |
|                        ^^^^^^^^^^^^^^^^^^                                                          |
| ValueError: invalid literal for int() with base 10: '18397,25781,36106'                            |
|                                                                                                    |
| Traceback (most recent call last):                                                                 |
|   File "/Users/orenmazor/Projects/Plex-Meta-Manager/plex_meta_manager.py", line 251, in start      |
|     config = ConfigFile(default_dir, attrs)                                                        |
|              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                                        |
|   File "/Users/orenmazor/Projects/Plex-Meta-Manager/modules/config.py", line 595, in __init__      |
|     self.Convert = Convert(self)                                                                   |
|                    ^^^^^^^^^^^^^                                                                   |
|   File "/Users/orenmazor/Projects/Plex-Meta-Manager/modules/convert.py", line 25, in __init__      |
|     self._mal_to_anidb[int(ids["mal_id"])] = anidb_id                                              |
|                        ^^^^^^^^^^^^^^^^^^                                                          |
| ValueError: invalid literal for int() with base 10: '18397,25781,36106'                            |
|                                                                                                    |
| invalid literal for int() with base 10: '18397,25781,36106'                                        |
|                                                                                                    |
|====================================================================================================|
```

digging around, it looks like this commit in https://github.com/meisnate12/Plex-Meta-Manager-Anime-IDs/commit/7a6e0f43c51f923730460c08e129d7daf7f3ac48 has converted some integers into a comma delimited string of integers, which this code isn't handling yet.

I'm not sure whether this is an oversight in the anime library or PMM itself not supporting this type of configuration

**Comment by revspm** [comment]
I am a new user setting up PMM - have this issue - confused me for a while...

**Comment by orenmazor** [comment]
> I am a new user setting up PMM - have this issue - confused me for a while...

I'm not an anime user so I just added a fast exit to the config setup for anime ids in modules/convert.py:
 
![Screenshot 2023-01-03 at 5 21 02 PM](https://user-images.githubusercontent.com/384279/210450901-dd35cf28-d433-4f43-9bcd-ea499502d8a4.png)

this is obviously not a real solution. I just didnt want this to block me today

**Comment by jakesmorrison** [comment]
I ran into this problem today. This issue is with the [anime json](https://raw.githubusercontent.com/meisnate12/Plex-Meta-Manager-Anime-IDs/master/pmm_anime_ids.json). 

`  "9826": {
    "tvdb_id": 267440,
    "mal_id": "18397,25781,36106"
  },
`

The mal_id is supposed to be an int and not a string. Either that needs to change or the convert.py code needs to be able to accept a comma separated string. 

I don't have anime, so I just removed that json url and hard coded a small part of the json. 
`
#self.config.get_json(anime_lists_url)

self._anidb_ids = {"1": {"tvdb_id": 72025,"mal_id": 290,"anilist_id": 290}}
`

**Comment by chazlarson** [comment]
This is fixed in nightly; 1.18.2 soon to be released will contain the fix as well.
